test/jdk/java/lang/Double/ParseDouble.java

```diff
@@ -759,6 +760,39 @@ private static void testStrictness() {
             throw new RuntimeException("Inconsistent conversion");
     }
 
+    private static void testFastPaths() {
+        /* Exercises the fast paths in jdk.internal.math.FloatingDecimal. */
+        check("1", 1.0);
```

Comment:
Something to consider: specify the double values in hex literals -- this would exercise entirely different code paths than the decimal ones.